templates/CRM/common/standalone.tpl

```diff
@@ -44,8 +44,8 @@
     {/if}
 
     {if $pageTitle}
-      <div class="crm-title">
-        <h1 class="title">{if $isDeleted}<del>{/if}{$pageTitle}{if $isDeleted}</del>{/if}</h1>
+      <div class="crm-page-title-wrapper">
+        <h1 class="crm-page-title">{if $isDeleted}<del>{/if}{$pageTitle}{if $isDeleted}</del>{/if}</h1>
```

Comment:
This seems worrying... are we sure `$isDeleted` variable is defined on literally every page? And also... why are we doing this?